client/src/lib/solana/sol-data.ts

```diff
@@ -38,6 +49,46 @@ export class SolData extends Assignable {
   assertionMethod: string[];
   service: ServiceEndpoint[];
 
+  constructor({
```

Comment:
I don't see any changes related to Assignable. Why is the explicit contractor required now (same question for DecentralizedIdentifier).

Reply:
There were quite a few type errors elsewhere because the constructor just allowed an types to be passed. By explicitly writing the types it eliminates those type errors.

Reply:
ok thanks!